Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -94,6 +96,13 @@ time of Cilium:
  * Helm options ``cluster.name`` and ``cluster.id``
  * Cilium CLI install options ``--set cluster.name`` and ``--set cluster.id``
 
+Example install using the Cilium CLI:
```

Comment:
I think it would make sense to link to the installation guide too https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/#cilium-quick-installation
as it captures more cases of how to install Cilium for different cloud providers etc and it probably makes sense to familiarize yourself with it before copying commands :)

Reply:
"See other use cases and familiarize yourself with the Cilium CLI".  Under the code example?

The thing is, for this install, users have to set the cluster.name and cluster.id, if we send them to the quick install, they might go through the install, and the validation, without setting those values.

I wanted to link to the other docs, I just wasn't sure which link wouldn't get them off track.

Maybe the link to the quick installation should go by this sentence?

"It is best to assign both these attributes at installation time of Cilium." (link to quick install)

Reply:
Added a link after the required install options, and before the code block.